### PR TITLE
PVAYLADEV-1139: Update build instructions

### PR DESCRIPTION
**Pull request by jhyoty**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,18 +51,14 @@ Submit a pull request to [X-Road source code Github repository](https://github.c
...
 ## X-Road installation
 
-**Local installation**
-
-X-Road central servers, configuration proxies and security servers can be automatically installed with the Ansible scripts found in ansible subdirectory. See instructions in [ansible/README.md](ansible/README.md)
-
-**Development installation**
-
-To setup a Virtualbox based development machine with X-Road servers running in LXD containers follow the instructions in [src/Vagrant.md](src/Vagrant.md)
-
```
Comment (iluwatar) [inline README.md:61]: Why was "Development installation" section removed?

File: ansible/hosts/lxd_hosts.txt [hunk 2]
```diff
@@ -2,25 +2,26 @@
...
-#configuration proxies (ubuntu lxd containers)
+#configuration proxies (ubuntu lxd containers, optional)
 [cp-servers]
-xroad-lxd-cp ansible_connection=lxd
+#xroad-lxd-cp ansible_connection=lxd
 
 #certification authority, time stamping authority and ocsp service server for testing purposes (ubuntu)
 [ca-servers]
-xroad-lxd-ca ansible_connection=lxd
+xroad-lxd-cs ansible_connection=lxd
```
Comment (iluwatar) [inline ansible/hosts/lxd_hosts.txt:11]: Should it be `xroad-lxd-ca`?

File: src/Vagrant.md [hunk 3]
```diff
@@ -1,48 +1,60 @@
...
 
-    Solution is to find out what is your VirtualBox Storage controller type and set it correctly. Alternatively you can try updating Virtualbox.
-    - ```vboxmanage showvminfo {virtualboxname}```
-    - Find line with ```Storage Controller Name (1):``` followed with name of Storage controller. Change line 27 of Vagrantfile to match that.
-        ```'--storagectl', '{Storage-Controller-Name}',```
+If you get `Stderr: VBoxManage: error: Could not find a controller named 'SCSI'` error on host machine, find out what is your VirtualBox Storage controller type and fix it in the Vagrantfile:
+
+`vboxmanage showvminfo xroad_dev | grep "Storage Controller Name (1)"`
+
+Change line 27 of Vagrantfile to match the contorller name (`'--storagectl', '{Storage-Controller-Name}' )
```
Comment (iluwatar) [inline src/Vagrant.md]: typo, should be controller

File: src/Vagrantfile [hunk 4]
```diff
@@ -66,16 +71,17 @@ Vagrant.configure("2") do |config|
...
      lxd init --auto --storage-backend=dir
      echo net.ipv4.ip_forward=1 >>/etc/sysctl.conf
      sysctl -p
+     sudo sed -i'' -r 's/(AcceptEnv*)/#\\1/' /etc/ssh/sshd_config
+     systemctl reload sshd
   SHELL
 
-  config.vm.provision "prepare_buildhost", type: "shell", path: "prepare_buildhost.sh", privileged: false
-
+  config.vm.provision 'prepare_buildhost', type: 'shell', path: 'prepare_buildhost.sh', privileged: false
```
Comment (iluwatar) [inline src/Vagrantfile]: Just wondering if `prepare_buildhost.sh` should be called here. If it should, why not call `update_ruby_dependencies.sh` also? The other option would be to leave this to the user and just follow the build instructions.

**Review by iluwatar — CHANGES_REQUESTED** [review]
Some minor things that should be addressed

File: README.md (hunk 1 above)
Comment (jhyoty) [reply README.md:61]: It is moved under the build instructions. It was somewhat unclear what is the real difference between "development installation" and "building X-Road".

File: ansible/hosts/lxd_hosts.txt (hunk 2 above)
Comment (jhyoty) [reply ansible/hosts/lxd_hosts.txt:11]: The ca role can be installed on the cs instance, avoiding one container --> saves some resources if the development machine is low on resources.

File: src/Vagrantfile (hunk 4 above)
Comment (jhyoty) [reply src/Vagrantfile]: No strong opinion here, could be removed or kept as is.

**Pull request merged**